fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -1448,18 +1449,17 @@ ACTOR static Future<Optional<Version>> readVersionProperty(Reference<BackupConta
 	}
 }
 
-Future<KeyRange> BackupContainerFileSystem::getSnapshotFileKeyRange(const RangeFile& file, Optional<Database> cx) {
+Future<KeyRange> BackupContainerFileSystem::getSnapshotFileKeyRange(const RangeFile& file, Database cx) {
```

Comment:
any reason we want to change signature of this method?

Reply:
I initially made this optional because i didn't require all callsites to pass a valid DB object but now i do so I changed the definition here